specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json

```diff
@@ -1375,7 +1375,7 @@
           "title": "The scale for the cluster by autoscale settings"
         }
       },
-      "description": "Specifies the scale settings for the cluster. At least one of manual and autoScale settings must be specified. Only one of manual and autoScale settings can be specified."
+      "description": "At least one of manual or autoScale settings must be specified. Only one of manual or autoScale settings can be specified. if autoScale settings are specified, the system automatically scales the cluster up and down (within the supplied limits) based on the pending jobs on the cluster."
```

Comment:
The word "if" at the start of the third sentence should be capitalized.

Reply:
done